doc/content/htgr/assembly/index.md

```diff
@@ -0,0 +1,661 @@
+which we name `thm_temp`. Finally, the MOOSE heat conduction module will send the heat
+flux to [!ac](THM), so we add a variable named `flux` that we will use to compute
+the heat flux using the [DiffusionFluxAux](https://mooseframework.inl.gov/source/auxkernels/DiffusionFluxAux.html)
+auxiliary kernel.
+
+!listing /htgr/assembly/solid.i
+  start=AuxVariables
+  end=Executioner
+
+We use functions to define the thermal conductivities. The material
+properties for the [!ac](TRISO) compacts are taken as volume averages of the various constituent
+materials. We will evaluate the thermal conductivity for the boron carbide as a
+function of temperature by using `t` (which *usually* is interpeted as time) as
+a variable to represent temperature. This is syntax supported
+by the [HeatConductionMaterials](https://mooseframework.inl.gov/source/materials/HeatConductionMaterial.html)
+used to apply these functions to the thermal conductivity.
+
+!listing /htgr/assembly/solid.i
+  start=Functions
+  end=Postprocessors
+
+We define a number of postprocessors for querying the solution as well as for
+normalizing the fission power and heat flux, to be described at greater
+length in [#n1].
+
+!listing /htgr/assembly/solid.i
+  block=Postprocessors
+
+For visualization purposes only, we add
+[LayeredAverages](https://mooseframework.inl.gov/source/userobject/LayeredAverage.html)
+for the fuel and block temperatures. These will average the temperature in layers
+oriented in the $z$ direction, which we will use for plotting axial temperature
+distributions. We output the results of these userobjects to CSV using
+[SpatialUserObjectVectorPostprocessors](https://mooseframework.inl.gov/source/vectorpostprocessors/SpatialUserObjectVectorPostprocessor.html) and by setting `csv = true` in the output.
+
+!listing /htgr/assembly/solid.i
+  start=UserObjects
+
+Finally, we specify a [Transient](https://mooseframework.inl.gov/source/executioners/Transient.html)
+executioner. Because there are no time-dependent kernels in this input file,
+this is equivalent in practice to using a [Steady](https://mooseframework.inl.gov/source/executioners/Steady.html)
+executioner, but allows you to potentially sub-cycle the MOOSE heat conduction solve
+relative to the OpenMC solve
+(such as if you wanted to converge the
+[!ac](CHT) fully inbetween data exchanges with OpenMC).
+
+!listing /htgr/assembly/solid.i
+  block=Executioner
+
+### Fluid Input Files
+
+The fluid phase is solved with [!ac](THM), and is described in the `thm.i` input.
+This input file is built using syntax specific to [!ac](THM) - we will only briefly
+cover this syntax, and instead refer users to the [!ac](THM) manuals for more information.
+First we define a number of constants at the beginning of the file and apply
+some global settings. We set the initial conditions for pressure, velocity,
+and temperature and indicate the fluid [!ac](EOS) object
```

Comment:
EOS is the name of the former system, still used by SAM only.
Should be called just fluid properties

Reply:
well actually doesnt matter here, just an acronym